engines/scumm/detection.cpp

```diff
@@ -1329,6 +1331,21 @@ SaveStateDescriptor ScummMetaEngine::querySaveMetaInfos(const char *target, int
 	return desc;
 }
 
+static const ExtraGuiOption scummComiObjectLabelsOption = {
+	_s("Show Object Line"),
+	_s("Show the names of objects at the bottom of the screen"),
+	"object_labels",
+	true
+};
+
+const ExtraGuiOptions ScummMetaEngine::getExtraGuiOptions(const Common::String &target) const {
```

Comment:
We have some requirement in this API that an empty target shall return *all* options. You will need to implement that too.